bindings/python/src/models.rs

```diff
@@ -23,7 +23,19 @@ impl Model {
         ))
     }
 
-    fn save(&self, folder: &str, name: &str) -> PyResult<Vec<String>> {
+    #[args(kwargs = "**")]
+    fn save(&self, folder: &str, kwargs: Option<&PyDict>) -> PyResult<Vec<String>> {
```

Comment:
This is probably breaking as it changes to a named parameter. I'm not sure it can be specified as positional then. What do you think of having this instead?
```rust
#[args(name=None)]
fn save(&self, folder: &str, name: Option<&str>) -> PyResult<Vec<String>> {
```